.github/workflows/tests.yml

```diff
@@ -57,8 +58,11 @@ jobs:
         with:
           ruby-version: "${{ matrix.ruby }}"
           bundler-cache: true
```

Comment:
We could drop the `actions/checkout@v3` line on 44, but then we'd have to set this to `false`. Then we would not having caching available in `scripts/test-gem-install`, but maybe that's okay?